modules/ROOT/pages/platform/metrics-integration.adoc

```diff
@@ -0,0 +1,430 @@
+____
+
+image::metrics_integration.png[]
+
+For details, see https://prometheus.io/docs/prometheus/latest/configuration/configuration/[Prometheus configuration reference^].
+
+[#start-prometheus]
+=== Start Prometheus
+
+[,shell]
+----
+./prometheus --config.file=prometheus.yml
+----
+
+[#test-that-metrics-are-fetched]
+=== Test that metrics are fetched
+
+Open http://localhost:9090 and enter a metric name or expression in the search field (ex. `neo4j_aura_cpu_usage`). After pressing enter, last metric values should show up.
+
+[#use-grafana]
+=== Use Grafana
+
+Install and configure Grafana, adding the endpoint of the Prometheus instance configured in the previous step as a data source. Create visualizations, dashboards and alarms based on Neo4j metrics.
+
+[#example-of-using-datadog]
+== Example of using Datadog
+
+* Get a Datadog account
+* Install a Datadog agent as described in Datadog documentation
+
+[#configure-an-endpoint-with-token-authentication]
+=== Configure an endpoint with token authentication
+
+Edit `/etc/datadog-agent/conf.d/openmetrics.d/conf.yaml` to be as follows.
+
+____
+*_NOTE:_* replace the placeholders `<ENDPOINT_URL>`, `<AURA_CLIENT_ID>` and `<AURA_CLIENT_SECRET>` with corresponding values from the previous steps.
+____
+
+[,yaml]
+----
+
+init_config:
+instances:
+  - openmetrics_endpoint: <ENDPOINT_URL>
+    metrics:
+      - neo4j_.*
+    auth_token:
+      reader:
+        type: oauth
+        url: https://api.neo4j.io/oauth/token
+        client_id: <AURA_CLIENT_ID>
+        client_secret: <AURA_CLIENT_SECRET>
+      writer:
+        type: header
+        name: Authorization
+        value: "Bearer <TOKEN>"
+----
+
+For details, see https://docs.datadoghq.com/agent/?tab=Linux[Datadog Agent documentation^] and https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml[configuration reference^].
+
+[#test-that-metrics-are-fetched-2]
+=== Test that metrics are fetched
+
+* `sudo systemctl restart datadog-agent`
+* watch `/var/log/datadog/*` to see if fetching metrics happens or if there are warnings regarding parsing the config
+* check in Datadog metric explorer to see if metrics appear (after a couple of minutes)
+
+[#programmatic-support]
+== Programmatic support
+
+[#aura-api-for-metrics-integration]
+=== Aura API for Metrics Integration
+
+* Aura API (_v1beta5_) now supports fetching metrics integration endpoints using:
+ ** endpoint `+/tenants/{tenantId}/metrics-integration+` (for tenant metrics)
+ ** JSON property `metrics_integration_url` as part of `+/instances/{instanceId}+` response (for instance metrics)
+* Reference: https://neo4j.com/docs/aura/platform/api/specification/?urls.primaryName=Aura%20v1beta5#/[Aura API Specification^]
+
+[#aura-cli-for-metrics-integration]
+=== Aura CLI for Metrics Integration
+
+* Aura CLI has a new subcommand for `tenants` command to fetch tenant metrics endpoint:
++
```

Comment:
Please add this or else the example won't get code highlighting in the generated HTML, nor a copy button.